devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy

```diff
@@ -510,5 +533,5 @@ def configure() {
         // "send 0x${device.deviceNetworkId} 1 1", "delay 1500",
     ]
 
-    return configCmds + refresh()
+    return configCmds + zigbee.onOffConfig(0, 300) + refresh()
```

Comment:
The `5 minutes` reporting interval was a requirement set by VD (TV-Project). We no longer have these restraints. Let's use the default reporting `onOffConfig()` interval specified in: 
https://github.com/PhysicalGraph/appengine-zigbee/blob/master/src/main/groovy/physicalgraph/zigbee/Zigbee.groovy#L507
Which is `10min`.